manifests/init.pp

```diff
@@ -580,6 +581,8 @@
   }
   validate_bool($purge_keys_real)
 
+  validate_bool($manage_service)
```

Comment:
Could you add the logic that allows for `'true'` as well as `true` ? See the section just above with purge_keys